packages/shared/src/__tests__/__examples__/parallel.ts

```diff
@@ -0,0 +1,71 @@
+import { createMachine } from "xstate";
```

Comment:
since you have added fixes for both invokes and entry actions - it would be great to have dedicated tests for both of those in combination with parallel states. At the moment only invoke case got its test here.

Reply:
I'll remove this test, this was copied from a user's submission.

Reply:
It was a good test though (if it would be slimmed down) :p resolving compound states recursively and resolving all child states of a parallel state is different. Right now they are both kinda resolved automatically using the same internal thing but we can't guarantee that this will always be the case. So it would be better to have separate tests here for both cases

Reply:
Re-added